### DISCUSSION TITLE
Cloudflare是被淦了吗？

### DISCUSSION THREAD

**Discussion by NyaiiSan** [share121/inter-knot#4855] [科技]
![截图 2024-09-14 14-04-00](https://github.com/user-attachments/assets/c29559df-4922-4db8-8d63-05cecff1f4f9) 

下午起来挂在Cloudflare的服务连不上了，Ping一下全红，这是被台风吹化了吗

**Comment by BuYaoHoYo** [comment]
有可能哦，可能是线路出现了问题，建议几天后再试试
